robot_number: 2
branch_number: 2
frontier_number: 4
robot1_x: 7.005
robot1_y: 9.24
robot1_yaw: 82.8
robot2_x: 35.492
robot2_y: 12.04
robot2_yaw: 180
branch1_x: 8.873
branch1_y: 12.04
branch2_x: 5.137
branch2_y: 12.04
frontier1_x: 4.67
frontier1_y: 12.04
frontier2_x: 9.34
frontier2_y: 0
frontier3_x: 32.69
frontier3_y: 9.8
frontier4_x: 30.355
frontier4_y: 12.04
